requirements.txt

```diff
@@ -1 +1,3 @@
 boto==2.40.0
+publicsuffixlist==0.4.0
+requests==2.10.0
```

Comment:
I'm not, but the `publicsuffixlist` module is even though it doesn't have its own `requirements.txt` nor `setup.py` to require it. So I had to include it here myself. 😢